app/src/common/shared/com/igalia/wolvic/ui/widgets/WindowWidget.java

```diff
@@ -665,6 +669,7 @@ private void showDownloadsPanel(boolean switchSurface) {
             if (mView == null) {
                 setView(mDownloads, switchSurface);
                 mDownloads.onShow();
+                mViewModel.setIsFindInPage(false);
```

Comment:
I know I'm being a bit annoying here :) but these are exactly the kind of bugs I was trying to prevent when I mentioned that having to update so many call sites every time we change a panel was very error-prone (as it's humanly impossible to have all those call sites in mind)

Reply:
It can of course be good to have some sort of method to have those called together, but by now we only have `mViewModel.setIsFindInPage(false);`, so later I guess we can have a method if this gets complex.